sys/linux/sys.txt

```diff
@@ -268,6 +268,7 @@ fchown(fd fd, uid uid, gid gid)
 fchownat(dirfd fd_dir, file ptr[in, filename], uid uid, gid gid, flags flags[at_flags])
 fallocate(fd fd, mode flags[fallocate_mode], off intptr, len intptr)
 faccessat(dirfd fd_dir, pathname ptr[in, filename], mode flags[open_mode], flags flags[faccessat_flags])
```

Comment:
Isn't the definition for `faccessat()` is wrong?  It doesn't actually have a `flags` argument; that's why `faccessat2()` was added.  So can you fix the `faccessat()` description too?

Reply:
Thx! I just changed that (I'm actually surprised to see that it's still able to compile with that wrong syscall declaration).